src/database/migrations/2023_10_06_023551_create_posts_table.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('posts', function (Blueprint $table) {
+            $table->id();
+            $table->timestamps();
+            $table->string('content');
+            $table->string('image')->nullable();
+            $table->foreignId('user_id')->constrained();
+        });
```

Comment:
create a foreign key with reference and an action on cascade:

Reply:
I have resolved this comment, sir. Thank you! 🙇